public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,239 @@
+openapi: 3.0.0
+info:
+  title: CTC Guarantee Balance
+  contact: {}
+  version: '2.0'
+servers:
+  - url: https://test-api.service.hmrc.gov.uk/customs/guarantees
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk/customs/guarantees
+    description: Production
+    variables: {}
+paths:
+  /customs/guarantees/:grn/balance:
+    post:
+      tags:
+        - customs
+      summary: Get a Balance
```

Comment:
To avoid unnecessary capitalisation of words and to keep API definition in line with with Test Support and Traders API definitions, use `balance` instead.